### ISSUE TITLE
Inherited fields aren't listed on the generated CRUD

### ISSUE THREAD

**Issue by cleydsonjr** [craigburke/lazybones-angular-grails#8]
If you have some kind of inheritance on the domain classes, the inherited fields are considered to build the CRUD.

**Comment by craigburke** [comment]
So domain classes that use inheritance don't include the inherited properties? 

I'm sure you're already aware but here's the part that figures out what fields are included:
https://github.com/craigburke/lazybones-angular-grails/blob/master/templates/subtmpl-module/lazybones.groovy#L165

**Comment by cleydsonjr** [comment]
Yes, the getDeclaredFields() method doesn't consider the fields declared on superclasses. Apparently we need to use some loop or recursion to get inherited Fields
